Part A - Basics/07_control_flow/examples/control_with_break_statement.py

```diff
@@ -1,5 +1,5 @@
 # Example 1: Break statement inside for loop
-val = input("Enter a number: ")
```

Comment:
**Rejected**. 

It doesn't protect the code anyhow and in case the user enters a symbol it will still throw an exception.